.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest]
+
+    steps:
+      - name: XCode version
+        run: xcode-select -p
+
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 200
+
```

Comment:
try adding:
```
run: git fetch --tags
```

Reply:
Or just go with 0. It's just 3 seconds. ;-)

Reply:
(... and not our bandwidth. Mwhahaha.)